extensions/amp-image-slider/amp-image-slider.md

```diff
@@ -56,6 +54,8 @@ The slider could also take 2 optional `div`s for labels, which are used to add e
 </amp-image-slider>
 ```
 
-By default, once the slider is loaded, the compared images will be separated by a vertical bar, with helpful arrow hints displayed at its center. User could press mouse button down or touch to move the slider to the position of the pointer, and could then move the pointer to drag the slider bar left and right. For the left image, only part that is left to the bar is displayed; similar for the right image to only display the right portion.
+Once the slider is loaded, the compared images are separated by a vertical bar, with helpful arrow hints displayed at the slider's center.
+
+Users can mouse down or touch to move the slider to the position of the pointer, and can then move the pointer to drag the slider bar left or right. For the left image, only the part that is left to the bar is displayed; similarly for the right image, to only display the right portion.
 
-If we specify `tabindex` on `amp-image-slider`, users are also allowed to move the slider using keyboard. Pressing down left and right arrow moves the slider bar one step towards the corresponding direction. Pressing Home would bring the slider to the center, PageUp and PageDown to the left and right end of the slider body.
+If you specify `tabindex` on the `amp-image-slider` element, users can navigate the slider with their keyboards. Pressing the  down, left, or right arrow moves the slider bar one step towards the corresponding direction. Pressing the Home key brings the slider to the center. Pressing the PageUp or PageDown keys moves to the left or right end of the slider body.
```

Comment:
`Pressing the  down, left, or right arrow`
There seems to be an extra space here, between "the" and "down"